src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -205,9 +208,7 @@ const TransactionDetailsComponent = (props: Props) => {
           exchangeAmount: { [defaultIsoFiat]: amountFiat }
         })
       })
-      .catch(error => {
-        showError(error)
-      })
+      .catch(showError)
```

Comment:
```ts
.catch((error: unknown) => { showError(error) }
```

This is necessary so TransactionDetailsScene will show up in the stack trace. Otherwise we'll have no idea where the error was thrown.